test/unit/app/models/republishing_event_test.rb

```diff
@@ -3,7 +3,7 @@
 class RepublishingEventTest < ActiveSupport::TestCase
   extend Minitest::Spec::DSL
 
-  describe "content_type" do
+  describe "#content_type" do
```

Comment:
I went for no `#` as it's a property/association rather than strictly a method, but not too fussed about this.

Reply:
Ah, I suppose we aren't actually testing the `content_type` accessor methods here, rather the validation of the columns, so I suppose it makes sense not to label these as testing their namesake methods